### PR TITLE
CEDS-2462 GDSify pages to include logout button

### PR DESCRIPTION
**Pull request by PauloPortugal**

### CODE REVIEW

File: app/views/error_template.scala.html [hunk 1]
```diff
@@ -14,16 +14,20 @@
...
-@govuk_wrapper(title = pageTitle, contentHeader = Some(contentHeader), mainContent = mainContent)
+@govukLayout(
+ title = Title(pageTitle),
+ useCustomContentWidth = true) {
+ <div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+   <h1 class="govuk-heading-xl" >@heading</h1>
+   <p class="govuk-body">@message</p>
+  </div>
+ </div>
```
Comment (MaciejRewera) [inline app/views/error_template.scala.html:32]: Is this really the case? I wouldn't expect error page has to be wider.

File: test/handlers/ErrorHandlerSpec.scala [hunk 2]
```diff
@@ -55,11 +56,11 @@ class ErrorHandlerSpec extends UnitSpec with Stubs with OptionValues {
 
     "standardErrorTemplate" in {
 
-      val result = errorHandler.standardErrorTemplate("Page Title", "Heading", "Message")(request).body
+      val result = errorHandler.standardErrorTemplate("title", "heading", "message")(request).body
```
Comment (MaciejRewera) [inline test/handlers/ErrorHandlerSpec.scala:59]: Nit: You could create constants and reference them here and on assertions.

File: app/views/error_template.scala.html (hunk 1 above)
Comment (PauloPortugal) [reply app/views/error_template.scala.html:32]: Yes that is the case.

Comment (PauloPortugal) [reply app/views/error_template.scala.html:32]: Well I mean if we want to maintain the same width as we do now we will have to go with that.

File: test/handlers/ErrorHandlerSpec.scala (hunk 2 above)
Comment (PauloPortugal) [reply test/handlers/ErrorHandlerSpec.scala:59]: @MaciejRewera what do you mean with that? Sorry :)

**Pull request merged**